### PR TITLE
profiles: Move PYTHON_SINGLE_TARGET to py3

### PR DESCRIPTION
**Pull request by SoapGentoo**

### CODE REVIEW

**Comment by mgorny** [comment]
Wait a bit with it. @radhermit is working on making pkgcheck detect non-working USE defaults, and I guess that'd be useful here.

**Comment by SoapGentoo** [comment]
@mgorny I need to rework the defaults anyhow, as we have to use the same default for stable and unstable.

**Comment by mgorny** [comment]
I've set it to recheck. When it's done, the report should have RequiredUseDefault results.

File: profiles/base/package.use [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+# David Seifert <soap@gentoo.org> (26 Mar 2017)
+# Only python 3.5 supported
+dev-util/kdevelop-python:5 python_targets_python3_5
 dev-util/kdevelop-python:5 python_single_target_python3_5
 
-# Michael Palimaka <kensington@gentoo.org> (19 Mar 2017)
-# only python 3 is supported
-app-editors/gedit-plugins python_single_target_python3_4
+# David Seifert <soap@gentoo.org> (26 Mar 2017)
+# These packages only support one python impl
```
Comment (mgorny) [inline profiles/base/package.use]: I have no clue why you're doing this.

**Comment by mgorny** [comment]
https://qa-reports.gentoo.org/output/gentoo-ci/c1ca9f304/output.html#dev-python/uvloop
https://qa-reports.gentoo.org/output/gentoo-ci/c1ca9f304/output.html#media-gfx/blender
https://qa-reports.gentoo.org/output/gentoo-ci/c1ca9f304/output.html#app-text/asciidoc
https://qa-reports.gentoo.org/output/gentoo-ci/c1ca9f304/output.html#app-text/wklej
https://qa-reports.gentoo.org/output/gentoo-ci/c1ca9f304/output.html#dev-util/kdevelop-python
https://qa-reports.gentoo.org/output/gentoo-ci/c1ca9f304/output.html#sci-chemistry/pdb-tools

**Comment by mgorny** [comment]
Two left after next pkgcheck upgrade:
https://qa-reports.gentoo.org/output/gentoo-ci/c1ca9f304/output.html#app-text/wklej
https://qa-reports.gentoo.org/output/gentoo-ci/2d592f5a1/output.html#dev-python/pygcrypt

**Comment by h4emp3** [comment]
Looks like the tooling did not detect each and every packet not working with python3_* as the single target. [Spice-gtk](https://bugs.gentoo.org/show_bug.cgi?id=605636) is one of those.

**Comment by tanzer** [comment]
Is this the reason for packages failing to emerge today? 

Like this one:

 * Package:    dev-db/postgresql-9.5.5
 * Repository: gentoo
 * Maintainer: pgsql-bugs@gentoo.org
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux linguas_de linguas_en nls pam python python_single_target_python3_4 python_targets_python2_7 python_targets_python3_5 python_targets_python3_6 readline server ssl userland_GNU zlib
 * FEATURES:   preserve-libs sandbox userpriv usersandbox

...

 * The implementation chosen as PYTHON_SINGLE_TARGET must be added
 * to PYTHON_TARGETS as well. This is in order to ensure that
 * dependencies are satisfied correctly. We're sorry
 * for the inconvenience.
 * ERROR: dev-db/postgresql-9.5.5::gentoo failed (setup phase):
 *   Build target (python3_4) not in PYTHON_TARGETS.

On my system, Python 3.4 was not installed until today (2.7, 3.5, and 3.6 are). In addition I added

    dev-db/postgresql python_single_target_python3_5

to /etc/portage/package.use but that seems to be ignored.

**Comment by mgorny** [comment]
Please file a bug. Pull requests are not support media.

**Pull request closed without merging**